server/formatter.js

```diff
@@ -81,10 +81,13 @@ function normalizeHtml(html) {
 
       const decoded = qs.unescape(redirectUrl)
       const [isDoc, docId] = decoded.match(/docs\.google\.com.+\/d\/([^/]+)/i) || []
+      const [deepLink = ''] = decoded.match(/(?<=heading([^/+]))(.*)/i) || []
 
-      const {path: libraryPath} = isDoc ? list.getMeta(docId) || {} : {}
+      const { path: libraryPath } = isDoc ? list.getMeta(docId) : {}
```

Comment:
It looks like a change was introduced here that no longer provides the fallback if the docId does not match as expected. This was resulting in an uncaught error when I hit it. Can we put back the fallback here? Otherwise this looks great.

```suggestion
      const { path: libraryPath } = isDoc ? list.getMeta(docId) || {} : {}
```